cluster/gce/win1803/smoke-test.sh

```diff
@@ -41,7 +40,7 @@
 # Override this to use a different kubectl binary.
 kubectl=kubectl
 linux_deployment_timeout=60
-windows_deployment_timeout=240
+windows_deployment_timeout=120
```

Comment:
just a query why we have reduced the windows_deployment_timeout from 240 to 120.

Reply:
Because four minutes is a long time :) Our clusters with Windows nodes have become stable enough that if test pods don't start within two minutes then it's probably a bug that we should file and fix.

Also, note that this just applies to pods we start from this smoke-test which we invoke manually on newly started clusters; it doesn't apply to any K8s continuous testing.